src/Nest/Search/Suggesters/Suggest.cs

```diff
@@ -14,5 +15,10 @@ public class Suggest
 		public string Text { get; internal set; }
 		[JsonProperty("options")]
 		public IEnumerable<SuggestOption> Options { get; internal set; }
+
+		public IEnumerable<CompletionOption<T>> GetCompletionOptions<T>()
+		{
+			return Options.Select(x => new CompletionOption<T>(x));
```

Comment:
@Mpdreamz fixed in [e17d7a5a5ce](https://github.com/tkirill/elasticsearch-net/commit/e17d7a5a5ce1c78bcac9b9c376a5884131faa03f#diff-4787c8ec5a0b0d0f96180eb10b40baf5L21)